backend/src/graphql/directive/isAuthorized.ts

```diff
@@ -13,15 +10,9 @@ const isAuthorized: AuthChecker<any> = async (
 ) => {
   if (context.token) {
     const decoded = decode(context.token)
-    if (decoded.sessionId && decoded.sessionId !== 0) {
-      const result = await apiGet(
-        `${CONFIG.LOGIN_API_URL}checkSessionState?session_id=${decoded.sessionId}`,
-      )
-      context.sessionId = decoded.sessionId
-      context.pubKey = decoded.pubKey
-      context.setHeaders.push({ key: 'token', value: encode(decoded.sessionId, decoded.pubKey) })
-      return result.success
-    }
+    context.pubKey = Buffer.from(decoded.pubKey).toString('hex')
```

Comment:
I just store the hex string for the pubKey in the context. I still have the buffer in the jwt itself but towards the application I expose the hex since its the goto-value used. That way we can save all those conversions.